content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -306,9 +302,9 @@ The functionality conflict between _filmic_ and _base curve_, to provide the nec
 
 The "display-referred" workflow, used by default, automatically activates the _base curve_ and orders the modules in the pipeline as they were for darktable 2.6 and older. As such, it will apply the non-linear transformation at the beginning of the pipeline, and all editing will then be done on non-linear RGB, as almost all photo editing software does. The only merit of this workflow is that it is what many photographers are used to, but it will cause many problems with color consistency and the credibility of some local filters (blurring or sharpening), especially in scenes with wide dynamics.
 
-The "scene-referred" workflow is designed around _filmic_, to offer the same level of functionality as the "display-referred" workflow. It consist of adding, by default, +0.75 EV of exposure (in _exposure_ module), to adjust the global brightness and mid-grey to a similar value to that used by most JPEGs produced by SLR cameras.
-**_New:_** the _exposure_ module will also compensate the exposure bias set on the camera (the EV compensation of the lightmeter), commonly used to prevent highlight clipping, for photographers who expose to the right of the histogram. This feature relies on reading the EXIF field `ExposureBiasValue` which must be correctly filled in the RAW file by the camera for it to work. Note: for Fuji RAWs, it will be necessary to add another +0.75 EV, for an overall correction of +1.45 EV, to compensate for their native underexposure.
-Next, _filmic_ is applied by default and its basic settings take into account the global exposure correction (including +0.75EV mid-grey correction and exposure bias). This allows to have a working base reasonably close to the camera JPEG, directly by opening the RAW in darktable. But like any default preset, it is designed to work properly 80% of the time, and additional manual adjustments will be required in the remaining 20%; in particular, some cameras apply a +0.5 EV correction and others a +1 EV, so it is illusory to try to satisfy everyone.
+The "scene-referred" workflow is designed around _filmic_, to offer the same level of functionality as the "display-referred" workflow. It consist of adding, by default, +0.5 EV of exposure (in _exposure_ module), to adjust the global brightness and mid-grey to a similar value to that used by most JPEGs produced by SLR cameras.
+**_New:_** the _exposure_ module will also compensate the exposure bias set on the camera (the EV compensation of the lightmeter), commonly used to prevent highlight clipping, for photographers who expose to the right of the histogram. This feature relies on reading the EXIF field `ExposureBiasValue` which must be correctly filled in the RAW file by the camera for it to work. Note: for Fuji RAWs, it will be necessary to add another +0.75 EV, for an overall correction of +1.25 EV, to compensate for their native underexposure.
```

Comment:
"add another" worked when previous line said "+0.75", but since that is no longer true here's my suggestion:
```suggestion
**_New:_** the _exposure_ module will also compensate the exposure bias set on the camera (the EV compensation of the lightmeter), commonly used to prevent highlight clipping, for photographers who expose to the right of the histogram. This feature relies on reading the EXIF field `ExposureBiasValue` which must be correctly filled in the RAW file by the camera for it to work. Note: for Fuji RAWs, it will be necessary to add aditional +0.75 EV, for an overall correction of +1.25 EV, to compensate for their native underexposure.
```